granary/reddit.py

```diff
@@ -112,6 +112,9 @@ def praw_to_object(self, thing, type):
       """
     obj = {}
 
+    if any([not hasattr(thing,a) for a in ['id','author','permalink']]):
```

Comment:
as mentioned in the o-d PR, mind using `getattr` inline instead? similar to a common pattern in granary of using `.get()` on JSON objects retrieved from other APIs so we don't assume any individual field is there. eg:

https://github.com/snarfed/granary/blob/cd70a876ce26836caae65ba702c715bc5d665b31/granary/twitter.py#L1159